contracts/TracerPerpetualSwaps.sol

```diff
@@ -478,12 +478,15 @@ contract TracerPerpetualSwaps is ITracerPerpetualSwaps, Ownable {
         int256 amountToTakeFromInsurance
     ) external override onlyLiquidation {
         address insuranceAddr = address(insuranceContract);
+        require(
+            balances[insuranceAddr].position.quote - amountToTakeFromInsurance >= 0,
+            "TCR: Insurance not funded enough"
+        );
```

Comment:
should this throw an error or should we instead just empty the insurance fund of everything its got, even if it holds less than the specified amount?